apps/internal/version/version.go

```diff
@@ -5,4 +5,4 @@
 package version
 
 // Version is the version of this client package that is communicated to the server.
-const Version = "0.8.1"
+const Version = "0.9.0"
```

Comment:
I assume `0.9.0` will be the "beta" release where we aim to stabilize the public API?

Reply:
Yes, that's the idea, basically an RC. `0.9.0` would be the last `0.x` and would have the stable API.